vms/platformvm/validators/manager.go

```diff
@@ -295,6 +304,9 @@ func (m *manager) makeSubnetValidatorSet(
 		primaryValidator, ok := primarySet[nodeID]
 		if !ok {
 			// This should never happen
+			m.log.Error(ErrMissingValidator.Error(),
```

Comment:
For _this_ check, I think it should be `"missing validator"` though - not `"missing valdiator set"`